code/scripting/api/libs/graphics.cpp

```diff
@@ -1519,6 +1519,7 @@ ADE_FUNC(createTexture, l_Graphics, "[number Width=512, number Height=512, enume
 	if(idx < 0)
 		return ade_set_error(L, "o", l_Texture.Set(texture_h()));
 
+	//Since creating a render target does not increase load count, we need to still do so even though we just created the texture
```

Comment:
Can you reword this comment?  "We still need to do so" is confusing, as the connection between the comment and the parameter (which is not specified and therefore uses its default value) requires an inference on top of another inference.